client/client.go

```diff
@@ -128,6 +128,10 @@ func New(opts ...Option) (*Client, error) {
 	dialOptions = append(dialOptions, grpc.WithUnaryInterceptor(authInterceptor.Unary()))
 	dialOptions = append(dialOptions, grpc.WithStreamInterceptor(authInterceptor.Stream()))
 
+	if options.MaxRecvMsgSize != 0 {
+		dialOptions = append(dialOptions, grpc.WithDefaultCallOptions(grpc.MaxCallRecvMsgSize(options.MaxRecvMsgSize)))
+	}
```

Comment:
Let's keep the `MaxCallRecvMsgSize` to make it easy for users to find features in the gRPC reference.

```suggestion
	if options.MaxCallRecvMsgSize != 0 {
		dialOptions = append(dialOptions, grpc.WithDefaultCallOptions(grpc.MaxCallRecvMsgSize(options.MaxCallRecvMsgSize)))
	}
```